.bazelignore

```diff
@@ -3,3 +3,9 @@ java/client/build
 java/server/build
 node_modules
 java/build/production
+.ijwb
+bazel-bin
+bazel-genfiles
+bazel-out
+bazel-selenium
+bazel-testlogs
```

Comment:
Why is this needed? Doesn't bazel need to know this information?

Reply:
* `.ijwb` is a folder created and used by IntelliJ IDEA
* The other folders are output folders created by Bazel during the build/test process. These are output folders that don't contain core project artifacts.
Without these "ignore" entries, Bazel and IntelliJ end up scanning these folders for project specifications and other build artifacts that they will never contain, resulting in warning messages and extending the time required to build the project for no purpose.

Reply:
the `.bazelignore` file is to specify:
> directories within the workspace that you want Bazel to ignore, such as related projects that use other build systems

I don't think these qualify as that.

As for IntelliJ, it isn't going to read the `.bazelignore` file to know what to index. You need to use its own "Mark directory as..." assignments.

Reply:
I reverted these changes, as they are tangential to the focus of this PR. The warning messages that are emitted when Bazel scans these folders don't affect the actual build output; these messages are just noise.